tools/Utilities/src/ViewModels/UtilityViewModel.cs

```diff
@@ -68,7 +67,9 @@ public UtilityViewModel(string exeName, IExperimentationService? experimentation
 
     private void Launch()
     {
+        var activityId = Guid.NewGuid();
         _log.Information($"Launching {_exeName}, as admin: {LaunchAsAdmin}");
+        TelemetryFactory.Get<ITelemetry>().Log("Utilities_UtilitiesLaunchEvent", LogLevel.Critical, new UtilitiesLaunchEvent(activityId, Title, LaunchAsAdmin, UtilitiesLaunchEvent.Phase.Start));

```

Comment:
Naming nit: is it still recommended to include `Event` explicitly in the event name? I know that this is the pattern across some parts of Dev Home at least, but I _thought_ we were trying to move away from that. Tagging @dkbennett.

Reply:
Not a nit.  Please add `_event`

Reply:
Event is not necessary in the event name, but it should be on the class name.